Makefile

```
@@ -1,83 +1,105 @@
-all: push
-
 VERSION = edge
 TAG = $(VERSION)
 PREFIX = nginx/nginx-ingress
-
+NGINX_VERSION = 1.19.3
 GOLANG_CONTAINER = golang:1.15
```

I think the following should work, and would speed up initial builds for those without any golang docker images?
```suggestion
GOLANG_CONTAINER = golang:1.15-alpine
```



```
$ docker images golang
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
golang              1.15-alpine         1de1afaeaa9a        7 days ago          299MB
golang              1.15                75605a415539        3 months ago        839MB
```